keylime/keylime_logging.py

```diff
@@ -5,7 +5,20 @@
 
 from keylime import config
 
-logging_config.fileConfig(config.get_config("logging"))
+try:
+    logging_config.fileConfig(config.get_config("logging"))
+except KeyError:
+    logging_barebones_config: Dict[Any, Any] = dict(
+        version=1,
+        formatters={"f": {"format": "%(asctime)s %(name)-12s %(levelname)-8s %(message)s"}},
+        handlers={"h": {"class": "logging.StreamHandler", "formatter": "f", "level": logging.DEBUG}},
+        root={
+            "handlers": ["h"],
+            "level": logging.DEBUG,
+        },
+    )
+
+    logging_config.dictConfig(logging_barebones_config)
```

Comment:
Maybe a more direct approach can be:

```python
...
except KeyError:
    logging.basicConfig(format="%(asctime)s %(name)-12s %(levelname)-8s %(message)s", level=logging.DEBUG)
```

Reply:
Nice, you're right, it is more direct (and thus better).